src/app/pages/viewing/messages/messages.component.css

```diff
@@ -20,71 +9,122 @@
   font-size: 1em;
 }
 
+/* Message Card */
+.card {
+  display: inline-block;
+}
+
+.message-card {
+  position: relative;
+  background-color: var(--light-grey);
+  width: 100%;
+  word-break: break-word;
+  max-height: 30em;
+  overflow: hidden;
+  cursor: pointer;
+}
+
 .table td {
   overflow-wrap: break-word;
 }
 
+/* Message Carousel Page */
 .page {
   background-color: var(--accent);
   box-shadow: 0 0.15em 0.25em 0.05em rgba(0, 0, 0, 0.175);
   color: white;
   height: 70vh;
-  padding: 3em 6em 3em 6em;
-  display: flex;
+  padding: 1.5em;
   overflow-y: auto;
 }
 
-.card-header-text {
+.carousel-controls {
+  display: flex;
+  justify-content: space-between;
+  padding: 1em 2em 1em 2em;
+}
+
+.carousel-controls-button {
+  color: var(--accent);
   font-weight: bold;
-  margin-top: 0.2em;
 }
 
-.card-header {
+.carousel-controls-button:hover {
+  text-decoration: none;
+}
+
+/* Message Header */
+.msg-header {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
+  line-height: 1em;
+  padding: 0.5em 0.5em 0.5em 1em;
 }
 
-.card-header-author {
+.msg-header-group {
   display: flex;
   flex-direction: row;
 }
 
-.carousel-header {
-  display: flex;
-  flex-direction: row;
+.msg-header-text {
+  font-weight: bold;
+  font-size: 0.9em;
+  margin: 0.2em 0 0 0.5em;
+  max-width: 10em;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.msg-header-date {
+  font-size: 0.7em;
+  margin: 0 0 0 0.5em;
 }
 
 .profile-pic-container-small {
   background: rgba(255, 255, 255, 0);
-  height: 2em;
-  width: 2em;
+  height: 1.7em;
+  width: 1.7em;
   border-radius: 100%;
   display: flex;
 }
 
-.card {
-  display: inline-block;
-}
-
-.message-card {
-  background-color: var(--light-grey);
-  width: 100%;
-  word-break: break-word;
-  max-height: 30em;
-  overflow-y: auto;
-}
-
 #no-messages {
   font-size: 2em;
   color: var(--light-grey);
 }
 
-.delete-button {
+ /* Remove the blue outline around the button when it's clicked */
```

Comment:
Change 'the button' to be more descriptive, I don't know which buttons you're talking about and I thought this might be a feature that should have been on main.css